meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn.service

```diff
@@ -9,4 +9,4 @@ RemainAfterExit=yes
 ExecStart=@BINDIR@/prepare-openvpn
 
 [Install]
-WantedBy=resin.target
+WantedBy=config-json.service
```

Comment:
The prepare-openvpn service needs to be rerun whenever the config-json.service is activated (ie whenever the config.json changes), which is precisely what we need, the issue we're fixing is that the prepare-openvpn service is never rerun when the keys are updated